Sources/DarkModeCore/DMTraitCollection.m

```diff
@@ -311,3 +316,22 @@ - (DMTraitCollection *)dmTraitCollection {
 - (void)dmTraitCollectionDidChange:(DMTraitCollection *)previousTraitCollection {}
 
 @end
+
+@interface UIWindow (DMTraitEnvironment)
+
+@end
+
+@implementation UIWindow (DMTraitEnvironment)
+
+- (void)dmTraitCollectionDidChange:(DMTraitCollection *)previousTraitCollection {
+  [super dmTraitCollectionDidChange:previousTraitCollection];
+
+  if (@available(iOS 13, *)) {
+    if (previousTraitCollection.userInterfaceStyle != self.dmTraitCollection.userInterfaceStyle && _windowThemeChangeHandler)
+      _windowThemeChangeHandler(self);
```

Comment:
If there are multiple windows (iPad multi-window), would the `themeChangeHandler()` be called multiple times?

Reply:
yes, it is called per window, I think set a new value to `overrideUserInterfaceStyle` will also invoke it. so a UIWindow parameter is passed in the block to distinguish between windows.

Reply:
I think we'd better to point this out explicitly in the method documentation, and explain what kind of functionality should go into this handler.

Reply:
Agreed, I'll tell the difference in detail in the section for setting up FluentDarkModeKit in readme

Reply:
updated.